javagony/NthFibonacciNumber.java

```diff
@@ -0,0 +1,39 @@
+package javagony;
+import javagony.exceptions.*;
+
+public class NthFibonacciNumber {
+    public static void main(String[] args) throws BooleanException {
+        final int number;
+        try {
+            number = Integer.parseInt(args[0]);
+        } catch (ArrayIndexOutOfBoundsException | NumberFormatException e) {
+            System.out.println("Please pass a number in the command-line");
+            return;
+        }
+        try {
+            NumericalComparison.lessThan(number, 0);
+        } catch (TrueException trueException) {
+            System.out.println("Please pass a positive number in the command-line");
+        } catch (FalseException falseException) {
+            System.out.println(fibonacci(number));
+        }
+
+    }
+
+    private static int fibonacci(int number) throws BooleanException {
```

Comment:
```suggestion
    private static int fibonacci(int number) {
```
I don't think this function actually throws any exceptions.